spec/defaults_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "helper"
+
+if SimpleCov.usable?
+  describe SimpleCov do
+    skip "requires the default configuration" if ENV["SIMPLECOV_NO_DEFAULTS"]
```

Comment:
(with other point where it is I mean this here) :)

Reply:
Same story here, I just wanted to make this resilient in case people run specs with that set, since the profiles won't exist in that case.